app/components/application/index.jsx

```diff
@@ -4,38 +4,41 @@ import Alert from 'react-s-alert';
 import Modal from 'containers/modal';
 import Navigation from 'components/navigation';
 import Footer from 'components/footer';
-import styles from './styles';
+import styles from './styles.css';
 
-const Application = ({ children, session, logout, signin, signup }) => {
+const Application = ({
+  children, session, logout, signin, signup,
+}) => {
   const { loggedIn, currentUser } = session;
 
   return (
-    <div className={ styles.layout }>
-      <main className={ styles.wrapper }>
+    <div className={styles.layout}>
+      <main className={styles.wrapper}>
         <Navigation
-          currentUser={ currentUser }
-          loggedIn={ loggedIn }
-          logout={ logout }
-          signin={ signin }
-          signup={ signup }
+          currentUser={currentUser}
+          loggedIn={loggedIn}
+          logout={logout}
+          signin={signin}
+          signup={signup}
         />
```

Comment:
BUT WHYYYY?????!?!?!?!!

Reply:
We need to change perception about this case. We should look at it as **interpolation**. A lot of languages uses interpolation without spaces inside interpolable section. E.g.:

Template literals in JS:
```js
const example = `test test ${myVar} test test`;
```

String interpolation in Ruby:
```ruby
puts "Hello, #{name}!"
```

Slim-lang template:
```
div class=user.role
```